influxdb3/management_cloud_decicated.go

```diff
@@ -93,10 +93,12 @@ func (d *CloudDedicatedClient) CreateDatabase(ctx context.Context, config *Cloud
 		return errors.New("database must not nil")
 	}
 
-	if d.client.config.Database == "" {
-		return errors.New("database name must not be empty")
+	if db.ClusterDatabaseName == "" {
+		if d.client.config.Database == "" {
+			return errors.New("database name must not be empty")
+		}
+		db.ClusterDatabaseName = d.client.config.Database
```

Comment:
Describe this behaviour in function documentation.

Reply:
Sure, but before doing it, I'm considering whether this fallback may be unexpected for users. Do you think this behavior should be left with documentation, or should we just drop it?

Reply:
Keep this fallback and update documentation.

Reply:
Addressed with eea7b2184131782a96cb8e37d95dffd44feb3d7d